src/utilities/swagger-decorator.const.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiExcludeEndpoint, ApiExcludeController } from '@nestjs/swagger';
+import { applyDecorators } from '@nestjs/common';
+import { getConfigValue } from '@us-epa-camd/easey-common/utilities';
+
+const env = getConfigValue('EASEY_QA_CERTIFICATION_API_ENV', 'local-dev');
+const disable = ['locv','development','testing'].includes(env) ? false : true;
```

Comment:
Should the first array item be "local-dev"?
```suggestion
const disable = ['local-dev','development','testing'].includes(env) ? false : true;
```

Reply:
Yes, you are right. I forgot to change to local-dev after testing.